pages/profile/collections/index.js

```diff
@@ -47,46 +62,121 @@ export default function Collections({ user, mutateUser }) {
         onClick={() => router.back()}
       />
       <IconButton
-        style="plus"
-        top="var(--gap-out)"
+        onClick={toggleMenu}
         right="var(--gap-out)"
-        onClick={toggleAddCollection}
+        top="var(--gap-out)"
+        style="Menu"
+        rotate={menuVisible}
       />
+      {menuVisible && (
+        <MenuContainer top="5rem" right="var(--gap-out)">
+          <UnstyledButton onClick={toggleAddCollection}>
+            <Plus width={15} height={15} />
+            Kochbuch hinzufügen
+          </UnstyledButton>
+          <UnstyledButton onClick={toggleEdit}>
+            <Pen width={15} height={15} />
+            Kochbücher bearbeiten
+          </UnstyledButton>
+        </MenuContainer>
+      )}
+      {isEditing && (
+        <ButtonContainer>
+          <DeleteButton onClick={handleDeleteCollection}>
+            <Trash width={15} height={15} />
+            Rezepte entfernen
+          </DeleteButton>
+          <DeleteButton onClick={toggleEdit}>
+            <XSmall width={15} height={15} />
+            abbrechen
+          </DeleteButton>
+        </ButtonContainer>
+      )}
+
       {addCollection && (
-        <StyledForm onSubmit={addNewCollection}>
-          <StyledInput name="collectionName" placeholder="Kochbuch Name" />
-          <AddButton type="submit">
-            <Plus width={20} height={20} />
-          </AddButton>
-        </StyledForm>
+        <NewCollection
+          user={user}
+          mutateUser={mutateUser}
+          setModal={toggleAddCollection}
+        />
       )}
       <CollectionWrapper>
         {!user.collections.length && "Du hast noch keine Kochbücher angelegt."}
         {user.collections.map((col, index) => (
```

Comment:
It makes the code a lot more readable, if you are explicit when naming parameters, arguments, etc.

```suggestion
        {user.collections.map((collection, index) => (
```